react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export default class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    this.setState({ isClicked: true });
+  }
+
+  render() {
+    if (this.state.isClicked) {
+      return (
+        <body>
+          <div className="container-menu">
+            <ul>
+              <h2>Menu</h2>
+              <li>About</li>
+              <li>Get started</li>
+              <li>Sign In</li>
+            </ul>
+          </div>
+        </body>
```

Comment:
Remember this gets placed into the `index.html` so it will already be inside of a body tag.  We pretty much never put a body tag directly in our JSX.

Reply:
Ok i got it. Thanks Scott!